### PR TITLE
style: avoid using `this` in template

### PR DESCRIPTION
**Pull request by eddybrando**
Fixes https://eslint.vuejs.org/rules/this-in-template.html to prevent https://github.com/Qiskit/qiskit.org/pull/1598 CI pipeline's lint stage from failing

### CODE REVIEW

File: components/ui/PageSection.vue [hunk 1]
```diff
@@ -10,13 +10,13 @@
       <div
         class="page-section__main-content"
         :class="{
-          'page-section__main-content_alone': !this.$slots.aside
+          'page-section__main-content_alone': !$slots.aside
```
Comment (y4izus) [inline components/ui/PageSection.vue:13]: I think `PageSection` is not used anymore. Could you double check and if that's the case, delete this file?

**Review by y4izus — APPROVED** [review]
Thank you very much for doing this!! just one check and we can merge :)

File: components/ui/PageSection.vue (hunk 1 above)
Comment (eddybrando) [reply components/ui/PageSection.vue]: Good catch!

I found in total 4 components not being used currently:

- components/ui/AppIcon.vue
- components/ui/MdContent.vue
- components/ui/PageSection.vue
- components/ui/SoftwareStack.vue

I deleted those as well 🙂

Comment (y4izus) [reply components/ui/PageSection.vue]: Great!! thank you!!

**Review by y4izus — APPROVED** [review]

**Pull request merged**